lib/dfp_helper.rb

```diff
@@ -39,13 +39,13 @@ def dfp_define_slot(_i, options = {})
 </script>
       END
     end
-    
+
     def dfp_helper_head(options={single_request:true})
       return unless dfp_helper_slots.size > 0
       o = dfp_helper_slots.collect{|i|
         _targeting = (i[:targeting]||[]).collect{|k,v| ".setTargeting(#{k.to_json}, #{v.to_json})"}.join
         _slot_name = (i[:slot_name].blank?)?"":"#{i[:slot_name]} = "
-        "#{_slot_name}googletag.defineSlot('#{i[:id]}', [#{i[:size].map(&:to_s).join(', ')}], '#{i[:div_id]}').addService(googletag.pubads())#{_targeting};"
+        "#{_slot_name}googletag.defineSlot('#{i[:id]}', [#{i[:size].map(&:to_s).join(', ')}], '#{i[:div_id]}').setCollapseEmptyDiv(true).addService(googletag.pubads())#{_targeting};"
```

Comment:
what i've seen is the collection of Slot objects is here
```
googletag.pubads().$
# eg, googletag.pubads().$[0].setCollapseEmptyDiv(true)
```
but based on the key name (`$`), i'm not sure if that is meant to be private